configure

```diff
@@ -16,10 +16,11 @@ fi
 ## Parse the args, only looking for -camldir
 ## We avoid using shift to keep "$@" intact
 
+cmd=$ocaml
 last=
 for i; do
    case $last in
-       -camldir|--camldir) cmd="$i/ocaml"; break;;
```

Comment:
@gares: Assuming a highest priority given to the uniformity policy, what would be your secondary priorities for the double-dash options that you introduced: to align on the one-dash usage for Coq or to embrace the diversity and support also a GNU-style convention for options?